<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DaysInfo</class>
 <widget class="QDialog" name="DaysInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>70</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="Country_DetailsForm">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="City_DetailsForm">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>70</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="Line" name="line_22">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="MondayLable_2">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Monday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_23">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Day1Temp_DetailsForm_2">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>20 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDay1ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="Line" name="line_25">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="TuesdayLable_2">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_26">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Day2Temp_DetailsForm_2">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>18 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDay2ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_27">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="Line" name="line_28">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="WednesdayLable_2">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_29">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Day3Temp_DetailsForm_2">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>22 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDay3ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_30">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="Line" name="line_31">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ThursdayLable_2">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Thursday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_32">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>19 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDay4ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_33">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <widget class="Line" name="line_34">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="FridayLable_2">
       <property name="font">
        <font>
         <family>Cambria</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Friday</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_35">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="font">
        <font>
         <family>IrisUPC</family>
         <pointsize>26</pointsize>
        </font>
       </property>
       <property name="text">
        <string>18 °C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mDay5ImageLable">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_36">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
